config/main.py

```diff
@@ -1312,6 +1312,54 @@ def naming_mode_alias():
     """Set CLI interface naming mode to ALIAS (Vendor port alias)"""
     set_interface_naming_mode('alias')
 
+#
+# 'syslog' group ('config syslog ...')
+#
+@config.group()
+@click.pass_context
+def syslog(ctx):
```

Comment:
It seems that the function name "syslog" conflicts with the module "syslog", causing functions calling logxxx failed. 
Is it possible to rename it?

Reply:
@stephenxs / @jleveque the ticket: https://github.com/Azure/sonic-utilities/issues/635